### PR TITLE
add json type support.

### PR DESCRIPTION
**Pull request by joy999**
Add the "jsonSupport" parameter to support changing the JSON-formatted field from "string" to "*gjson.Json".

### CODE REVIEW

File: commands/gen/gen_dao.go [hunk 1]
```diff
@@ -34,6 +35,7 @@ type generateDaoReq struct {
 	JsonCase           string // JsonCase specifies the case of generated 'json' tag for model struct, value from gstr.Case* function names.
 	DirPath            string // DirPath specifies the directory path for generated files.
 	StdTime            bool   // StdTime defines using time.Time from stdlib instead of gtime.Time for generated time/date fields of tables.
+	JsonSupport        bool   // JsonSupport defines using *gjson.Json instead of string for geerated json fields of tables.
```
Comment (gqcn) [inline commands/gen/gen_dao.go:38]: 1. `geerated` -> `generated`
2. `JsonSupport` -> `GJsonSupport` for clear.

File: commands/gen/gen_dao.go [hunk 2]
```diff
@@ -73,6 +75,7 @@ OPTION
                          | Kebab           | any-kind-of-string |
                          | KebabScreaming  | ANY-KIND-OF-STRING |
     -/--stdTime          use time.Time from stdlib instead of gtime.Time for generated time/date fields of tables.
+	-/--jsonSupport      use jsonSupport to use *gjson.Json instead of string for geerated json fields of tables.
```
Comment (gqcn) [inline commands/gen/gen_dao.go:78]: 1. `geerated` -> `generated`
2. Use spaces instead of indent character in the help content.

**Review by gqcn — CHANGES_REQUESTED** [review]

File: commands/gen/gen_dao.go (hunk 1 above)
Comment (joy999) [reply commands/gen/gen_dao.go:38]: -jsonSupport参数要不要也一起改成-gJsonSupport

File: commands/gen/gen_dao.go (hunk 2 above)
Comment (joy999) [reply commands/gen/gen_dao.go:78]: Fixed

File: commands/gen/gen_dao.go (hunk 1 above)
Comment (gqcn) [reply commands/gen/gen_dao.go]: @joy999 对，改一下，保持命名统一。另外代码由冲突，你pull一下master。

Comment (joy999) [reply commands/gen/gen_dao.go]: 改好了，也重新pull合并过了。

**Pull request merged**